receiver/memcachedreceiver/testdata/fake_stats.json

```diff
@@ -0,0 +1,103 @@
+          "cas_badval":"0",
+          "cas_hits":"0",
+          "cas_misses":"0",
+          "cmd_flush":"0",
+          "cmd_get":"0",
+          "cmd_meta":"0",
+          "cmd_set":"0",
+          "cmd_touch":"0",
+          "conn_yields":"0",
+          "connection_structures":"3",
+          "crawler_items_checked":"0",
+          "crawler_reclaimed":"0",
+          "curr_connections":"2",
+          "curr_items":"0",
+          "decr_hits":"0",
+          "decr_misses":"0",
+          "delete_hits":"0",
+          "delete_misses":"0",
+          "direct_reclaims":"0",
+          "evicted_active":"0",
+          "evicted_unfetched":"0",
+          "evictions":"0",
+          "expired_unfetched":"0",
+          "get_expired":"0",
+          "get_flushed":"0",
+          "get_hits":"0",
+          "get_misses":"0",
+          "hash_bytes":"524288",
+          "hash_is_expanding":"0",
+          "hash_power_level":"16",
+          "incr_hits":"0",
+          "incr_misses":"0",
+          "libevent":"2.1.12-stable",
+          "limit_maxbytes":"67108864",
+          "listen_disabled_num":"0",
+          "log_watcher_sent":"0",
+          "log_watcher_skipped":"0",
+          "log_worker_dropped":"0",
+          "log_worker_written":"0",
+          "lru_bumps_dropped":"0",
+          "lru_crawler_running":"0",
+          "lru_crawler_starts":"1",
+          "lru_maintainer_juggles":"61",
+          "lrutail_reflocked":"0",
+          "malloc_fails":"0",
+          "max_connections":"1024",
+          "moves_to_cold":"0",
+          "moves_to_warm":"0",
+          "moves_within_lru":"0",
+          "pid":"1",
+          "pointer_size":"64",
+          "read_buf_bytes":"49152",
+          "read_buf_bytes_free":"16384",
+          "read_buf_count":"3",
+          "read_buf_oom":"0",
+          "reclaimed":"0",
+          "rejected_connections":"0",
+          "reserved_fds":"20",
+          "response_obj_bytes":"16384",
+          "response_obj_count":"1",
+          "response_obj_oom":"0",
+          "round_robin_fallback":"0",
+          "rusage_system":"0.023452",
+          "rusage_user":"0.033109",
+          "slab_global_page_pool":"0",
+          "slab_reassign_busy_deletes":"0",
+          "slab_reassign_busy_items":"0",
+          "slab_reassign_chunk_rescues":"0",
+          "slab_reassign_evictions_nomem":"0",
+          "slab_reassign_inline_reclaim":"0",
+          "slab_reassign_rescues":"0",
+          "slab_reassign_running":"0",
+          "slabs_moved":"0",
+          "threads":"4",
+          "time":"1633116932",
+          "time_in_listen_disabled_us":"0",
+          "total_connections":"4",
+          "total_items":"0",
+          "total_malloced":"0",
+          "touch_hits":"0",
+          "touch_misses":"0",
+          "unexpected_napi_ids":"0",
+          "uptime":"12",
+          "version":"1.6.9"
+       },
+       "Slabs":{
+          
+       },
+       "Items":{
+          
+       }
+    }
+ }
```

Comment:
json and yaml files must end with a newline. Anywhere you see this needs to be fixed: 
![image](https://user-images.githubusercontent.com/5255616/141339022-6ec4f41b-004c-4fed-b195-365147644628.png)

Reply:
Updated